tests/test_config_flow.py

```
@@ -0,0 +1,39 @@
+"""Tests for Garmin Connect config flow."""
+
+from unittest.mock import patch
+
+from homeassistant import config_entries
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResultType
+
+from custom_components.garmin_connect.const import DOMAIN
+
+
+async def test_form_user(hass: HomeAssistant, _mock_garmin_client) -> None:
```

Ruff (0.14.10)

Fixture `_mock_garmin_client` without value is injected as parameter, use `@pytest.mark.usefixtures` instead

(PT019)